styles.css

```diff
@@ -915,7 +924,7 @@ nav:hover {
     background: #555;
   }
 
-  .mobile-container .Project_work .workform4 button {
+  .mobile-container .Project_work .workform4 .button4 {
```

Comment:
- [ ] Your popup must follow the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, tests, margins,...)  kindly make this requested changes: 
- [ ] Your popup image should have 100% width instead of `width: 90%;` and make sure your image is perfectly centering to the center (you need to remove the margin: 0 60px; and instead you can use text-align, margin: 0 auto, justify-content: center....) kindly make this 👍🏻  👨🏻‍🎨
- [ OPTIONAL] It would be better if the background is not scrollable when the popup page is open. you can use the `overflow:hidden;` CSS property to avoid this. 😄 👨🏻‍🎨 🎨 


Your Implementation | Figma Design
:---:|:---:
![sobad](https://user-images.githubusercontent.com/79058364/175521788-af321fc5-e0c7-49b9-bbab-d1902bb568ad.png) | ![style1](https://user-images.githubusercontent.com/79058364/173584118-d3329953-5244-41d7-8d3e-500b9bc68ada.png)